egs/aspire/s5/local/chain/tuning/run_blstm_7b.sh

```diff
@@ -183,6 +209,8 @@ if [ $stage -le 12 ]; then
     --egs.chunk-width $chunk_width \
     --egs.chunk-left-context $chunk_left_context \
     --egs.chunk-right-context $chunk_right_context \
+    --egs.chunk-left-context-initial=0 \
```

Comment:
please add these options to prep_test_aspire.sh below and modify the script to pass them through. the defaults in the script can be -1.  If you don't do this, the decoding will be mismatched.